package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "vote.gov",
+  "version": "1.0.0",
```

Comment:
I'm not 💯 on the version for this, but there needs to be a version here in order to have `gulp` properly display when executing the `gulp *` commands.